package.json

```diff
@@ -12,24 +12,28 @@
   "bugs": {
     "url": "https://github.com/WalletConnect/walletconnect-utils/issues"
   },
+  "workspaces": [
+    "browser/*",
+    "jsonrpc/*",
+    "crypto/*",
+    "misc/*",
+    "relay/*"
+  ],
   "scripts": {
-    "clean": "lerna clean --yes",
-    "bootstrap": "lerna bootstrap",
```

Comment:
>  '-   "bootstrap": "lerna bootstrap",'
I think we should remove the step from the main `README`

Reply:
Good callout, adding to my docs rewrite todos above 👍